### DISCUSSION TITLE
Getting the LS_PARAMETERS folder

### DISCUSSION THREAD

**Discussion by GGDLW** [NASA-LIS/LISF#1105] [General]
Hello,

I recently downloaded LIS (and the components) and it appears that I don't have any input land surface data I can use to run LDT. Would anyone know where and how can I find the LS_PARAMETERS folder?

**Comment by Gayathridevi123** [comment]
Hi @GGDLW,

There is a public LIS Data Portal that contains various datasets of meteorological forcing, parameters, and test cases for use with LDT, LIS, and LVT (https://portal.nccs.nasa.gov/lisdata_pub/). Please check here https://portal.nccs.nasa.gov/lisdata_pub/data/ for met forcing and parameters. Test cases are here https://portal.nccs.nasa.gov/lisdata_pub/TESTING/testcases/data/. 

Hope this will help you.
Gayathri
